### DISCUSSION TITLE
how override vendor resource

### DISCUSSION THREAD

**Discussion by Giovanni-Petrella** [filamentphp/filament#5397] [Help]
How can I override a vendor's resource? I want to do it of the filament-email-log plugin, I tried to make a class that extends the plugin resource but I get two menu items: 

use Ramnzys\FilamentEmailLog\Filament\Resources\EmailResource as BaseEmailResource;

class PrismaEmailResource extends BaseEmailResource
{

    protected static ?string $navigationIcon = 'heroicon-o-mail';

......
![Screenshot 2023-01-05 at 19-48-49 Emails - PrismASD](https://user-images.githubusercontent.com/70906593/210857726-6e792fcd-0dba-4f1b-84ad-f63e1e5dbb20.png)

**Comment by danharrin** [comment]
It's the plugin maintainer's responsibility to allow you to not register their version of the resource, via config or something

**Reply by Giovanni-Petrella** [reply]
So if the developer doesn't provide this option, I can't do anything?

**Reply by danharrin** [reply]
You can use the Advanced Navigation API to manually redefine all navigation items

https://filamentphp.com/docs/2.x/admin/navigation#advanced-navigation-customization

**Reply by Giovanni-Petrella** [reply]
Ok thanks, but at this point I'd better incorporate the extension rather than rewrite the whole navigation bar since this option is not necessary for me at the moment.
